dev-requirements.txt

```diff
@@ -121,6 +121,9 @@ more-itertools==4.3.0 \
     --hash=sha256:c187a73da93e7a8acc0001572aebc7e3c69daf7bf6881a2cea10650bd4420092 \
     --hash=sha256:c476b5d3a34e12d40130bc2f935028b5f636df8f372dc2c1c01dc19681b2039e \
     --hash=sha256:fcbfeaea0be121980e15bc97b3817b5202ca73d0eae185b4550cbfce2a3ebb3d
+mouseinfo==0.1.2 \
```

Comment:
Looks like for mac support, we'll need to add `pyobjc-core` and `pyobjc` as dependencies (see `mouseinfo`'s `install_requires` [here](https://github.com/asweigart/mouseinfo/blob/master/setup.py#L33-L34))

Reply:
I pushed another commit, which shows example of platform dependent dependency. But, this also means we will have to different requirements.txt files based on the environment. I have added one for Linux, you will have to add one for mac. Details are in https://github.com/jazzband/pip-tools#cross-environment-usage-of-requirementsinrequirementstxt-and-pip-compile